src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -823,6 +834,21 @@ public Visitable visit(ObjectCreationExpr newExpr, Void p) {
     return super.visit(newExpr, p);
   }
 
+  /**
+   * Converts a qualified class name into a relative file path.
+   *
+   * @param qualifiedName The qualified name of the class.
```

Comment:
Qualified names shouldn't have angle brackets. Either change the documentation here to say that angle brackets for type variables are permitted, or remove lines 845-847 (I recommend the former).